thirdparty/openapi3/components/schemas/ThirdpartyRequestsTransactionsIDAuthorizationsPostRequest.yaml

```diff
@@ -1,3 +1,4 @@
+# TODO: should this be removed? 
```

Comment:
Yes. We can prune these resources/components at a later date.

Reply:
Cool.